Task-6/js/index.js

```diff
@@ -83,11 +83,26 @@ function validFormFieldInput(event) {
       return false;
     }
     _taskManager.addTask(taskValue, descriptionValue, assigneeValue, dueValue);
-    console.log(_taskManager.tasks);
+    // console.log(_taskManager.tasks);
     _taskManager.render();    
   }
   
   
 
-  submitElement = document.getElementById('submit-form');
+  submitElement = document.querySelector('#submit-form');
   submitElement.addEventListener("click", validFormFieldInput);
+
+  
+  let renderList = document.querySelector('#render-task-list');
+  renderList.addEventListener("click", (event) => {
+    // console.log(event.target.classList);
+    if(event.target.classList.contains('done-button')) {
+      let parentTask = event.target.parentElement.parentElement.parentElement;
```

Comment:
Everything works as expected, however on line 100 of index.js chaining .parentElement's to traverse the DOM could break our code if the structure of our card changes in the future -- adopting the .closest() method is an alternative approach ( for example if the button moves to a different node  ) 
My suggestion is this:
`100 let parentTask = event.target.closest('li')`

Reply:
Love that observation! Just committed the recommended implementation.